CHANGELOG.md

```diff
@@ -4,4 +4,5 @@ Yii Framework 2.1 CAPTCHA extension Change Log
 1.0.0 under development
 -----------------------
 
+- Enh #1: Add Docs Use yii\captcha\CaptchaValidator in yii 2.1 #10 (cjtterabyte)
```

Comment:
Docs improvements should not be mentioned in changelog - it is not a real change and adds only unnecessary noise.